uber/server.py

```diff
@@ -145,20 +146,25 @@ def register_jsonrpc(service, name=None):
     assert name not in jsonrpc_services, '{} has already been registered'.format(name)
     jsonrpc_services[name] = service
 
+
 jsonrpc_handler = _make_jsonrpc_handler(jsonrpc_services, precall=jsonrpc_reset)
 cherrypy.tree.mount(jsonrpc_handler, join(c.PATH, 'jsonrpc'), c.APPCONF)
 
-
-def reg_checks():
-    sleep(600)  # Delay by 10 minutes to give the system time to start up
-    check_unassigned()
-    detect_duplicates()
-    check_placeholders()
-
-# Registration checks are run every six hours
-DaemonTask(reg_checks, interval=21600, name="mail reg checks")
-
-DaemonTask(SendAllAutomatedEmailsJob.send_all_emails, interval=300, name="send emails")
-
-# TODO: this should be replaced by something a little cleaner, but it can be a useful debugging tool
-# DaemonTask(lambda: log.error(Session.engine.pool.status()), interval=5)
+uber.scheduler.register_task(
+    lambda: uber.scheduler.schedule_N_times_per_day(4, check_unassigned),
+    category="reports"
+)
```

Comment:
I like this overall approach to registering tasks, but it's a little weird to have to pass the call to ``schedule_N_times_per_day`` through a lambda.

You could write a utility function to wrap this which would allow you to just say:

```python
schedule_task(check_unassigned, times_per_day=4, category='reports')
```

A naive implementation would just dispatch to the underlying function like this:

```
def schedule_task(func, times_per_day, category):
    uber.scheduler.register_task(
        lambda: uber.scheduler.schedule_N_times_per_day(times_per_day, func),
        category=category
    )
```

I would recommend something like that, because these 4 calls are a little copy/pastey and I think a call like

```python
schedule_task(check_unassigned, times_per_day=4, category='reports')
```

is just as clear while being less verbose.